test/lint/lint-python.sh

```diff
@@ -81,14 +89,6 @@ enabled=(
     W606 # 'async' and 'await' are reserved keywords starting with Python 3.7
 )
 
-if ! command -v flake8 > /dev/null; then
```

Comment:
Does this cause any problems?

I think I prefer leaving it here; to have the list of exceptions (a "constant") above the code as much as possible.

Reply:
LGTM.

Reply:
> I think I prefer leaving it here; to have the list of exceptions (a "constant") above the code as much as possible.

Agree. Fixed.